tests/e2e/testUtils.go

```diff
@@ -0,0 +1,242 @@
+package e2e
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/tebeka/selenium"
+	"github.com/tebeka/selenium/chrome"
+	"github.com/tebeka/selenium/firefox"
+	"log"
+	"mime"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+)
+
+const FIREFOX string = "Firefox"
+const CHROME string = "Chrome"
+
+var currentPath, _ = filepath.Abs(".")
+var DownloadPath = filepath.Join(currentPath, "tmp")
+var seleniumPort = 4444
+var server *http.Server
+var MaxWaitTimeout = 90 * time.Second
```

Comment:
Initialize all these values in `setUp()` and handle the error returned with `filepath.Abs(".")`